convertCSV.js

```diff
@@ -1,34 +1,47 @@
 class CsvPhraseImporter {
 
   phrases = []
+
+  fileName = ""
   
   constructor(event) {
     this.readFile(event)
   }
   
   readFile(event) {
+
+    // Since there is only one file able to be inputted at a time, index 0 is the only array index
     let file = event.target.files[0]
+    this.fileName = event.target.files[0].name
     
     const reader = new FileReader();
     reader.addEventListener('load', (event) => {
-      //console.log(csv)
       this.csvToPhrase(event.target.result)
     });
     reader.readAsText(file);
   }
   
   csvToPhrase(result) {
     let rawParseResults = Papa.parse(result)
+
+    /*
+      Note: gets each *column* from each *row*
+    */
     rawParseResults.data.forEach((column) => {
```

Comment:
I think you have 'column' mixed up with 'row'.  Typically, you iterate through the rows, then access the data for the row by the column index.

Reply:
I was thinking of them as numbered columns, but when thinking about 'for each' loops this might be better wording.